main/webapp/modules/core/scripts/project.js

```diff
@@ -436,20 +436,29 @@ Refine.postProcess = function(moduleName, command, params, body, updateOptions,
     }
   }
 
-  Refine.setAjaxInProgress();
+  var runChange = function() {
+    Refine.setAjaxInProgress();
+
+    Refine.postCSRF(
+        "command/" + moduleName + "/" + command + "?" + $.param(params),
+        body,
+        onDone,
+        "json"
+    );
 
-  Refine.postCSRF(
-    "command/" + moduleName + "/" + command + "?" + $.param(params),
-    body,
-    onDone,
-    "json"
-  );
+    window.setTimeout(function() {
+        if (!done) {
+        dismissBusy = DialogSystem.showBusy();
+        }
+    }, 500);
+  }
```

Comment:
Indentation looks off here

```suggestion
    window.setTimeout(function() {
      if (!done) {
        dismissBusy = DialogSystem.showBusy();
      }
    }, 500);
  }
```